### PR TITLE
Quick nav: Consider tab headers as form fields (#10432)

### LINKED ISSUE #10432: ARIA Tabs not accessible as forms controls
ARIA Tabs are not accessible using the shortcut "F" or in the lists of form fields.

### Steps to reproduce:
Open in either Chrome or Firefox the following page : 
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

### Actual behavior:
Tabs are not accessible using the shortcut "F" or in the lists of form fields

### Expected behavior:
Tabs are focused with the "F" key and are listed in the form fields

#### NVDA installed/portable/running from source:
NVDA installed 

#### NVDA version:
2019.2.1

#### Windows version:
Windows 10 version 1803 - OS Build 17134.1069 

#### Name and version of other software in use when reproducing the issue:
Firefox 69.0.3 64 Bits (FR-fr locale)
Chrome Version 77.0.3865.120 (Official build) (64 bits) (FR-fr locale)

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

### PR DESCRIPTION
**Pull request by JulienCochuyt**
### Link to issue number:

Fixes #10432

### Summary of the issue:

Tab headers are currently not considered when navigating by form fields.

### Description of how this pull request fixes the issue:

Add the tab header roles to those considered as form fields:
- UIAHandler.UIA_TabItemControlTypeId
- oleacc.ROLE_SYSTEM_PAGETAB

### Testing performed:

Tested from source on Windows 10 version 1903 (18362.418) with:
- Google Chrome 78.0.3904.70 (64 bits)
- Mozilla Firefox 70.0 (64 bits)
- Microsoft Edge (latest from OS)
- Microsoft Internet Explorer 11 (latest from OS)

### Known issues with pull request:

The diff looks bigger only to comply with linting.
Effective changes are really just adding the relevant role in three lists for gecko_ia2, MSHTML and UIA.

### Change log entry:

Section: New features or Changes

Tab headers are now considered as form fields when using quick navigation with the letter "f".

### CODE REVIEW

**Comment by dingpengyu** [comment]
hi @JulienCochuyt 
The following conflicts:
Conflicting files
source/UIABrowseMode.py
thanks

**Comment by JulienCochuyt** [comment]
@dingpengyu: Fixed. Thanks!

**Review by LeonarddeR — DISMISSED** [review]

**Review by feerrenrut — DISMISSED** [review]
The changes look fine.

I understand that this PR has been waiting a long time, but I have a small doubt about whether tabs SHOULD be considered form fields. After all you can't enter information with them. I suppose they could be used in place of a radio button, but that seems pretty broken.

I suspect the reason this has been waiting so long is because others are also uncertain, along with the diff being a little hard to follow (due to necessary reformatting).

I'd like to flag this decision, and defer to @michaelDCurran for an opinion about whether this fits.

**Comment by feerrenrut** [comment]
After talking to @michaelDCurran about this, we are happy to go ahead. We also treat buttons as form fields.

**Review by feerrenrut — APPROVED** [review]
